docs/snippets/common/ghp-github-action.yml.mdx

```diff
@@ -27,7 +27,11 @@ jobs:
       # Set up Node
       - uses: actions/setup-node@v4
         with:
-          node-version: '16.x'
+          node-version: '20.x'
+
+      # If using Yarn Modern, you also need to add the following action to install your configured Yarn version:
+      # - name: Install Yarn
+      #   run: corepack enable && yarn install
```

Comment:
@esimkowitz If you're ok with it, could you move the Corepack step above? Before the node setup action, turning it into something as follows:

```yml
name: "Build and Publish Storybook to GitHub Pages"

on:
  push:
    branches:
      - 'your-branch-name' # Replace with the branch you want to deploy from

permissions:
  contents: read
  pages: write
  id-token: write

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v4
      # Enable the steps below if you're working with Corepack's supported package managers (e.g. Yarn 2+, pnpm)
      #- name: Enable Corepack
      #  run: corepack enable
      - uses: actions/setup-node@v4
        with:
          node-version: 20
      - run: yarn install
      - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.3
        with:
          install_command: yarn install # default: npm ci
          build_command: yarn build-storybook # default: npm run build-storybook
          path: storybook-static # default: dist/storybook
          checkout: false # default: true
```

Reply:
I think corepack is bundled with Nodejs, I haven't tested if it's available before NodeJS gets installed

Reply:
@esimkowitz I've tested it, and corepack is already bundled with Node.js, so we don't need it here.